catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcDatabaseOperations.java

```diff
@@ -77,6 +77,12 @@ public List<String> listDatabases() {
     }
   }
 
+  protected ResultSet getDatabase(Connection connection, String databaseName) throws SQLException {
```

Comment:
the name is a little confusing, seems like load a database

Reply:
It indeed are loading a database for MySQL, what do you mean?

Reply:
Yes,it's expected for MySQL, for PG, we did use the database name actually.

Reply:
> I saw you get multi schemas from `getDatabase`, could you move following code to getSchema?
> 
> ```java
>       while (resultSet.next()) {
>         if (Objects.equals(resultSet.getString(1), databaseName)) {
>           found = true;
>           break;
>         }
>       }
> ```

I can't get it clearly, you mean I should move it to the `getSchema` NOT `listSchema` method?